lib/state/state.js

```diff
@@ -11,5 +14,9 @@ export default function(self) {
     ascending: self.opts.orderBy && self.opts.orderBy.hasOwnProperty('ascending')?self.opts.orderBy.ascending:true
   };
 
+  if (typeof self.$store.state[self.name]!=='undefined') {
+   return merge(true, state, self$store.state[self.name])
```

Comment:
You have an error here:

`self$store.state[self.name]`

should be

`self.$store.state[self.name]`